wsmaster/che-core-api-devfile/src/main/resources/schema/devfile.json

```diff
@@ -221,6 +221,48 @@
             "description": "Describes whether projects sources should be mount to the tool. `CHE_PROJECTS_ROOT` environment variable should contains a path where projects sources are mount",
             "default": "false"
           },
+          "command": {
```

Comment:
Please consider adding the corresponding test cases to `DevfileSchemaValidatorTest` (I mean that dockerimage is valid if command/args is string/array